### ISSUE TITLE
Display a count of items, eg "(23) Bookmarks tagged coffee"

### ISSUE THREAD

**Issue by ajft** [ckolderup/postmarks#119]
GUI improvement to display the count of items for each tag

**Comment by steve-bate** [comment]
I have tag counts on [my instance](https://bookmarks.stevebate.dev/) (fork). Is this what you had in mind?

**Comment by andypiper** [comment]
That definitely looks like what I'd imagine (and remember from sites like del.icio.us) and what is described in the issue title). I see you have a few other nice additions as well @steve-bate   - the Atom "radar" icon for individual tag pages, and the +/- filtering - good work!

**Comment by ckolderup** [comment]
So I mean no offense by it, I think it's pure preference, but this is the first time someone's requested something that I don't personally want, which I knew would happen eventually!

I do appreciate that Steve's version of it has appropriate markup that I could use in a custom stylesheet to hide it 😄 I always preferred Pinboard's approach of making the tag color lighter/darker instead of adding lots of numbers to the page.

I'd definitely merge a PR from Steve (or someone reproducing how Steve implemented it), and it will motivate me to get us to a place where people can more easily reset/add to styles in a way that's compatible with using git to pull in new changes without overwriting CSS customizations.

Thanks for the report and I hope you can take my commentary in the spirit I mean it (you're obviously not alone in wanting this!)

**Comment by ckolderup** [comment]
yeah, I could even see there being a little dropdown menu at the corner of the tags list itself that lets you toggle it on the fly? That requires more client-side JS and probably a cookie or localStorage or something... Probably a user setting makes the most sense.
